backend/src/main/resources/db/migration/V1_0_6__Change_Organization_Employees_Table_Name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE organization_users RENAME TO organization_employees
```

Comment:
Merge the content into v1_0_4. The same way we deliver one commit (squashed) for the feature, we should deliver only one script for the feature as well so that it is easier to track history and rollback in case of a disaster.